components/elm/src/data_types/ColumnDataType.F90

```diff
@@ -1479,9 +1501,35 @@ subroutine col_ws_init(this, begc, endc, h2osno_input, snow_depth_input, watsat_
          ptr_col=this%h2osoi_ice, l2g_scale_type='veg')
 
     this%h2osoi_ice(begc:endc,:) = spval
-        call hist_addfld2d (fname='SOILICE_ICE',  units='kg/m2', type2d='levgrnd', &
-        avgflag='A', long_name='soil ice (ice landunits only)', &
-        ptr_col=this%h2osoi_ice, l2g_scale_type='ice')
+    call hist_addfld2d (fname='SOILICE_ICE',  units='kg/m2', type2d='levgrnd', &
+         avgflag='A', long_name='soil ice (ice landunits only)', &
+         ptr_col=this%h2osoi_ice, l2g_scale_type='ice')
+
+    if (use_polygonal_tundra) then
+      this%iwp_subsidence(begc:endc)    = spval
+      this%iwp_ddep(begc:endc)          = spval
+      this%iwp_exclvol(begc:endc)       = spval
+      this%iwp_microrel(begc:endc)      = spval
+      this%frac_melted(begc:endc,:)     = spval
+      this%excess_ice(begc:endc,:)      = spval
+
+      call hist_addfld2d (fname='EXCESS_ICE', units = '1', type2d='levgrnd', &
+           avgflag='A', long_name='Excess ground ice (0 to 1)', &
+           ptr_col=this%excess_ice, l2g_scale_type='veg') ! <- RPF: should this be natveg?
```

Comment:
Can delete `! <- RPF: should this be natveg?`

Reply:
Removed comment